app/models/agents/java_script_agent.rb

```diff
@@ -114,12 +114,9 @@ def execute_js(js_function, incoming_events = [])
       context["getOptions"] = lambda { |a, x| interpolated.to_json }
       context["doLog"] = lambda { |a, x| log x }
       context["doError"] = lambda { |a, x| error x }
-      context["getMemory"] = lambda do |a, x, y|
```

Comment:
Yeah, I agree, but there's currently no one-argument use for the function and this is meant to be a minimal fix.  We can make further improvements later.